content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -30,117 +30,122 @@ bare metal and the clusters.
+      name: tls
+      protocol: TLS
+    tls:
+      mode: AUTO_PASSTHROUGH
+    hosts:
+    - "*.local"
+    {{< /text >}}    
 
-1.  Install `root-cert.pem` under `/var/run/secrets/istio/`.
+Applying this gateway will route any of the traffic from the VM destined for the workloads in the mesh running on `*.local`
 
-1.  Install `cluster.env` under `/var/lib/istio/envoy/`.
+At this point, you can continue with the [Setup Virtual Machine documentation](/docs/setup/install/virtual-machine/). 
 
-    {{< text bash >}}
-    $ sudo cp cluster.env /var/lib/istio/envoy
-    {{< /text >}}
+## Verify setup
 
-1.  Transfer ownership of the files in `/etc/certs/` , `/var/lib/istio/envoy/` and `/var/run/secrets/istio/`to the Istio proxy.
+After setup, the machine can access services running in the Kubernetes cluster
+or on other VMs. When a service on the VM tries to access a service in the mesh running on Kubernetes, the endpoints (ie, IPs) for those services will be the ingress gateway on the Kubernetes Cluster. To verify that, on the VM run the following command (assuming you have a service named `httpbin` on the Kubernetes cluster):
 
     {{< text bash >}}
-    $ sudo chown -R istio-proxy /etc/certs /var/lib/istio/envoy /var/run/secrets/istio/
+$ curl -v localhost:15000/clusters | grep httpbin
     {{< /text >}}
 
-1.  Start Istio using `systemctl`.
+This should show endpoints for `httpbin` that point to the ingress gateway similar to this:
 
-    {{< text bash >}}
-    $ sudo systemctl start istio
+    {{< text text >}}
+outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_active::1
+outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_connect_fail::0
+outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::cx_total::1
+outbound|8000||httpbin.default.svc.cluster.local::34.72.46.113:443::rq_active::0
     {{< /text >}}
 
-## Send requests from VM workloads to Kubernetes services
-
-After setup, the machine can access services running in the Kubernetes cluster
-or on other VMs.
+The IP `34.72.46.113` in this case is the ingress gateway public endpoint.    
 
-The following example shows accessing a service running in the Kubernetes cluster from a VM using
-`/etc/hosts/`, in this case using a service from the [Bookinfo example](/docs/examples/bookinfo/).
+## Send requests from VM workloads to Kubernetes services
 
-1.  Connect to the cluster service from VM as in the example below:
+At this point we should be able to send traffic to `httpbin.default.svc.cluster.local` and get a response from the server. You may have to set up DNS in `/etc/hosts` to map the `httpbin.default.svc.cluster.local` domain name to an IP since the IP will not resolve. In this case, the IP should be an IP that gets routed to the local Istio Proxy sidecar. You can use the IP from the `ISTIO_SERVICE_CIDR` variable in the `cluster.env` file you created in the [Setup Virtual Machine documentation](/docs/setup/install/virtual-machine/).
```

Comment:
future: we should document alternative DNS setups probably. Or keep waiting for DNS interception I guess, may land in 1.8